docs/jaeger-query-v2-setup.md

```diff
@@ -0,0 +1,56 @@
+# Jaeger Query v2 Setup Guide
```

Comment:
I don't think this file is necessary. The documentation changes should be made on the docs website.